Personalizing your Skywallet.md

```diff
@@ -0,0 +1,110 @@
+# Personalizing your Skywallet
+
+After completing the initialization of your Skywallet, you can start to personalize your Skywallet using the Hardware Configuration window in the desktop wallet.
+
+Upon configuring your Skywallet using your desktop wallet, the Skywallet will be listed in the Wallet list.
+
+The list can be viewed when you open your Desktop wallet.
+
+<Screenshot of the list of wallets displayed when the desktop wallet is opened. One of the wallets in the list needs to be a Skywallet>
+
+To personalize your Skywallet, you have to click on the **"Hardware Wallet"** button.
+
+<Screenshot of the desktop wallet highlighting the Hardware wallet button>
+
+Once you click on this button, the following window will appear:
+
+<Screenshot of the Hardware configuration window with the options available when the Skywallet is used after configuration for the first time>
+
+During your first time use of the Skywallet, you will find these two warnings associated with **"Creating a Backup"** and **"Setting a Pin"** being displayed in the Desktop wallet window.
+
+Both of these warnings are to be dealt with seriously or it may compromise the security of your Skywallet.
+
+In order to complete these two tasks, you can use the options in the Hardware Configuration window
+
+Following are the options available in this window:
+
+**Create a backup**
+
+**Create PIN code**
+
+**Wipe the device**
+
+## Create a backup
+
+The Seed of your Skywallet is a key component. Hence, once the initialization is complete it is important to backup this Seed and store it in a secure location.
+
+To backup the Seed of your skywallet, click on the button **"Create a backup"**.
+
+<Screenshot of the hardware configuration window with an arrow pointing towards the create a backup button or highlighting the button>
+
+Once you click the button, a warning message(?) will be displayed.
+
+After accepting the same the Skywallet will start to display the words in your Seed one by one.
```

Comment:
Replace `same` with actual window or button name

Reply:
Replaced this part, however, there is no clarity about what the warning message is. That is why I have put a question mark in the bracket.